cmd/apmtool/tracegen.go

```diff
@@ -75,3 +73,29 @@ func suffixString(s string) string {
 	}
 	return fmt.Sprintf("%s-%s", s, string(b))
 }
+
+// NewTraceGenCmd returns pointer to a Command that generates distributed tracing data using go-agent and otel library
+func NewTraceGenCmd(commands *Commands) *cli.Command {
+	return &cli.Command{
+		Name:   "tracegen",
+		Usage:  "",
```

Comment:
Should this contain a description for the command?

Reply:
Oops yes.